ls1intum/Artemis
# `Iris`: Chatbot sends the first message when there is no chat history

### Checklist
#### General


#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


Chatbot always send the first message.
### Description

When a user first open Iris chatbot, the chatbot will send out the first message, say hello and briefly explain what it does. After the user chat with the chatbot, when he/she opens the chatbot for the second time, the chat history will be shown in the chat window without the first hello message from the chatbot
### Steps for Testing

Prerequisites:
- 1 Student
- 1 new Programming Exercise(no chat history) with Iris enabled

1. Log in to Artemis
2. Navigate to the programming exercise
3. Open the chat window, you are supposed to see one message from chatbot.
4. Talk to chatbot, then close the chat window and exit the programming exercise page or refresh this exercise page
5. Navigate to the programming exercise and open the chat window, you are supposed to see your chat history without the first hello message from the chatbot

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/60965100/0a7a2aab-21b9-47d5-8d94-e02d1262ea04)
![image](https://github.com/ls1intum/Artemis/assets/60965100/48d8745a-17eb-4364-9519-44d652afd712)
![image](https://github.com/ls1intum/Artemis/assets/60965100/03f950dd-3d07-44c9-9fad-c06e58f3e7bf)
